src/main/java/org/gridsuite/study/server/repository/StudyCreationRequestRepository.java

```diff
@@ -24,6 +24,8 @@
 @Repository
 public interface StudyCreationRequestRepository extends JpaRepository<StudyCreationRequestEntity, UUID> {
 
+    List<StudyCreationRequestEntity> findAllByUserId(String userId);
```

Comment:
is it worth it to define this, instead of the findByPrimaryKey ? What's the difference